daffodil-runtime1/src/main/scala/org/apache/daffodil/dpath/DFDLXFunctions.scala

```diff
@@ -110,13 +162,47 @@ case class DFDLXXor(recipes: List[CompiledDPath], argType: NodeInfo.Kind) extend
       case NodeInfo.UnsignedShort => arg1.getInt ^ arg2.getInt
       case NodeInfo.UnsignedByte => (arg1.getShort ^ arg2.getShort).toShort
           //$COVERAGE-OFF$
-      case _ => Assert.invariantFailed(s"dfdlx:XOR not supported")
+      case _ => Assert.invariantFailed(s"dfdlx:bitXOR not supported")
```

Comment:
This error message should be dfdlx:bitXor to match the function name that we actually expect.